pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -1242,6 +1244,57 @@ var _ = Describe("HyperconvergedController", func() {
 				Expect(r.ensureIMSConfig(hco, newReq())).ToNot(BeNil())
 			})
 		})
+
+		Context("Vm Import", func() {
+			It("should create if not present", func() {
+				hco := &hcov1alpha1.HyperConverged{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      name,
+						Namespace: namespace,
+					},
+					Spec: hcov1alpha1.HyperConvergedSpec{},
+				}
```

Comment:
BeforeEach, like this:
``` golang
Context("Vm Import", func() {

	var hco *hcov1alpha1.HyperConverged
	var req *hcoRequest

	BeforeEach(func() {
		hco = newHco()
		req = newReq(hco)
	})
	It("should create if not present", func() { ...
```
then remove the hco var from `It("should create if not present", func() ...` and `It("should find if present", func()...`

Reply:
I think we can do it a bit differently and keep your changes.

Reply:
> I think we can do it a bit differently and keep your changes.

This code: 
``` golang
hco := &hcov1alpha1.HyperConverged{
		ObjectMeta: metav1.ObjectMeta{
		Name:      name,
		Namespace: namespace,
	},
	Spec: hcov1alpha1.HyperConvergedSpec{},
}
```
Repeated itself many times. the `newHco()` method do the same. This test file is huge - let's save code line when possible.